aict-MERN/aict-site-w-react/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,34 @@
+import { useRef, useState } from 'react'
+import './navbar.css' 
+import logo from './images/dcrdlogo-1.png'
+
+const Navbar = () => {
+    const [styleDisplay, setStyleDisplay] = useState('0')
+    const noOtherPagesMessage = useRef();
+
+    const questionMarkOnMouseEnter = () =>{
+        setStyleDisplay('1')
+    }
+    const questionMarkOnMouseLeave = () =>{
+        setStyleDisplay('0')
+    }
+
+    return (
+        <>
+            <header>
+                <img src={logo} alt="" />
+                <h2 className='question-mark' onMouseEnter={questionMarkOnMouseEnter} onMouseLeave={questionMarkOnMouseLeave}>?</h2>
+                <div className="no-other-pages-message" style={{"opacity": styleDisplay}} ref={noOtherPagesMessage}>
```

Comment:
the css attribute `visibility: hidden` does what you're trying to do with the opacity here. once again, consider replacing that with a ternary